playbooks/common/openshift-cluster/upgrades/v3_1_to_v3_2/pre.yml

```diff
@@ -100,8 +100,8 @@
   hosts: oo_masters_to_config:oo_nodes_to_config
   vars:
     target_version: "{{ '1.2' if deployment_type == 'origin' else '3.1.1.900' }}"
-    openshift_docker_hosted_registry_insecure: True
-    openshift_docker_hosted_registry_network: "{{ hostvars[groups.oo_first_master.0].openshift.common.portal_net }}"
+    docker_hosted_registry_insecure: "{{ openshift_docker_hosted_registry_insecure | default(True) }}"
```

Comment:
> IMO we should remove the registry insecure setting from playbooks and maintain the openshift_docker_hosted_registry_network setting in playbooks with the openshift_docker* prefix.

We still need a way to compute it, and prior to the deployment of the registry. If we defer the setting of it, then we are looking at an additional docker restart on provisioning.